project2/styles.css

```
@@ -0,0 +1,194 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed:300|Roboto:300,400,500,900&display=swap');
+
+* {
+    box-sizing: border-box;
+}
+
+.u-button-reset {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+.clearfix::before,
+.clearfix::after {
+    content: ' ';
+    display: table;
+}
+
+.clearfix::after {
+    clear: both;
+}
+
+body {
+    color: rgb(66, 30, 35);
+    font-family: 'Roboto Condensed', sans-serif;
+    background-color: rgb(88, 52, 56);
+}
+
+h2 {
+    font-family: 'Roboto', sans-serif;
+    font-weight: 500;
+    font-size: 19.5px;
+    line-height: 16.px;
+    position: relative;
+    margin-bottom: 50px;
+}
+
+p {
+    font-family: 'Roboto Condensed', sans-serif;
+    font-size: 10.5px;
+    line-height: 16.5px;
+    font-weight: light;
+    color: rgb(88, 52, 56);
+    border-top: 0.5px solid rgb(88, 52, 56);
+    padding-top: 5px;
+    letter-spacing: 0.6px;
+    margin-top: 3px;
+}
+
+
+.container {
+    background-color: rgb(245, 245, 232);
+    height: 714.483px;
+    /* width: 548px; */
+    position: relative;
+    border-top-right-radius: 55.5px;
+    border-top-left-radius: 55.5px;
+    border-bottom-right-radius: 55.5px;
+    box-shadow: -3px 0px 10px rgba(0, 0, 0, 40%),
+        /* above, -3px to the left, 0px to the top 10px spread */
+        3px 0px 10px rgba(0, 0, 0, 40%),
+        /* above, 3px to the right, 0px to the top 10px spread */
+        12px 0px 24px rgba(0, 0, 0, 40%),
+        /* above, 12px to the right, 0px to the top 24px spread */
+        0px 12px 24px rgba(0, 0, 0, 40%);
+    /* above, 0px horizontal, 12px to the bottom 24px spread */
+    min-width: 500px;
+    max-width: 548px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+h3 {
+    display: inline;
+    /* position: relative;
+    bottom: -22.5px; */
+}
+
+.short {
+    font-family: 'Roboto Condensed', sans-serif;
+    font-size: 10.5px;
+    font-weight: 300;
+    color: rgb(122, 122, 59);
+}
+
+.cost {
+    font-family: 'Roboto', sans-serif;
+    font-weight: 900;
+    font-size: 17.2px;
+    color: rgba(109, 82, 82, 74%);
+    float: right;
+    margin: 0;
+}
+
+.logo {
+    width: 157.3px;
+    height: 49.1835px;
+    position: absolute;
+    top: 31.2px;
+    left: 29.7px;
+    z-index: 1;
+}
+
+.product {
+    position: relative;
+    float: bottom;
+    width: 100%;
+    height: 548px;
```

Since you've set the width to be a percent, you can use the `auto` value here to keep your image dimensions correct:
```suggestion
    height: auto;
```